plugins/catalog/src/components/EntitySwitch/EntitySwitch.tsx

```diff
@@ -33,6 +33,7 @@ export interface EntitySwitchCaseProps {
     entity: Entity,
     context: { apis: ApiHolder },
   ) => boolean | Promise<boolean>;
+  default?: boolean;
```

Comment:
I don't think we should add this to the public surface. It's superfluous; if there's no `if`, it's a default rendered thing. Even now that we have the possibility of rendering multiple items, the defaults should still be easily discernible by the absence of `if`. If you want to refactor the private interfaces below then that's fine.

Reply:
Removed it from this public interface

Reply:
@drodil You seem to have forgotten to actually remove it.

Reply:
Argh, true. Will remove.